projects/assets/src/translations/en/common.ts

```diff
@@ -22,6 +22,7 @@ export const common = {
     add: 'Add',
     breadcrumbs: 'breadcrumbs',
     selectFile: 'Select file',
+    clear: 'Clear',
```

Comment:
I see the rest oof the labels are in the cart lib.
Is this one label here because it is generic enough too be reused by other features?

Reply:
Yes that was my reasoning